```
2024-06-28 09:23:24 ----------------Check results--------------------
2024-06-28 09:23:24 paddle.vision.datasets.Flowers:1
2024-06-28 09:23:24 <string>:9: error: Unused "type: ignore" comment  [unused-ignore]
2024-06-28 09:23:24 <string>:30:22: error: Item "Image" of "Union[Tensor, Image, ndarray[Any, dtype[Any]]]" has no attribute "shape"  [union-attr]
2024-06-28 09:23:24 Found 2 errors in 1 file (checked 1 source file)
2024-06-28 09:23:24 
2024-06-28 09:23:24 
2024-06-28 09:23:24 >>> Mistakes found in type checking!
2024-06-28 09:23:24 >>> Please recheck the type annotations. Run `tools/type_checking.py` to check the typing issues:
```

加错地方了，注意下行号

--- a/python/paddle/vision/datasets/flowers.py
+++ b/python/paddle/vision/datasets/flowers.py
@@ -90,7 +90,7 @@ class Flowers(Dataset):
             >>> for i in range(5):  # only show first 5 images
             ...     img, label = flowers[i]
             ...     # do something with img and label
-            ...     print(type(img), img.size, label)  # type: ignore
+            ...     print(type(img), img.size, label)
             ...     # <class 'PIL.JpegImagePlugin.JpegImageFile'> (523, 500) [1]
 
             >>> transform = T.Compose(
@@ -114,7 +114,7 @@ class Flowers(Dataset):
 
             >>> for img, label in itertools.islice(iter(flowers_test), 5):  # only show first 5 images
             ...     # do something with img and label
-            ...     print(type(img), img.shape, label)
+            ...     print(type(img), img.shape, label) # type: ignore
             ...     # <class 'paddle.Tensor'> [3, 64, 96] [1]
     """
 

